mmv1/third_party/terraform/services/resourcemanager/resource_google_service_account_test.go

```diff
@@ -88,6 +88,48 @@ func TestAccServiceAccount_basic(t *testing.T) {
 	})
 }
 
+// Test the option to ignore ALREADY_EXISTS error from creating a service account.
+func TestAccServiceAccount_createIgnoreAlreadyExists(t *testing.T) {
+	t.Parallel()
+
+	accountId := "a" + acctest.RandString(t, 10)
+	uniqueId := ""
+	displayName := "Terraform Test"
+	desc := "test description"
+	project := envvar.GetTestProjectFromEnv()
+	expectedEmail := fmt.Sprintf("%s@%s.iam.gserviceaccount.com", accountId, project)
+	acctest.VcrTest(t, resource.TestCase{
+		PreCheck:                 func() { acctest.AccTestPreCheck(t) },
+		ProtoV5ProviderFactories: acctest.ProtoV5ProviderFactories(t),
+		Steps: []resource.TestStep{
+			// The first step creates a basic service account
+			{
+				Config: testAccServiceAccountBasic(accountId, displayName, desc),
+				Check: resource.ComposeTestCheckFunc(
+					resource.TestCheckResourceAttr(
+						"google_service_account.acceptance", "project", project),
+					resource.TestCheckResourceAttr(
+						"google_service_account.acceptance", "member", "serviceAccount:"+expectedEmail),
+				),
+			},
+			{
+				ResourceName:      "google_service_account.acceptance",
+				ImportStateId:     fmt.Sprintf("projects/%s/serviceAccounts/%s", project, expectedEmail),
+				ImportState:       true,
+				ImportStateVerify: true,
+			},
+			// The second step creates a new resource that duplicates with the existing service account.
+			{
+				Config: testAccServiceAccountCreateIgnoreAlreadyExists(accountId, displayName, desc),
```

Comment:
Oh- this will need to have both blocks! It will technically work almost all of the time as-is because the create and delete will happen simultaneously, but we can eliminate the flake by adding `resource "google_service_account" "acceptance"` in the second config too.

(deletes may cause errors though as we'll then try to delete the resource twice- I can't think of a great way to work around that w/o adding new user-facing functionality to the resource)

Reply:
I see, thank you for the pointer.

I added both resources in the config block, and added a NOT_FOUND handling for DeleteServiceAccount.